pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -6027,6 +6027,26 @@ func TestChangefeedErrors(t *testing.T) {
 		`CREATE CHANGEFEED FOR foo into $1 WITH compression='gzip'`,
 		`kafka://nope`)
 
+	sqlDB.ExpectErrWithTimeout(
+		t, `required column idk not present on table foo`,
+		`CREATE CHANGEFEED FOR foo into $1 WITH headers_json_column_name='idk'`,
```

Comment:
These validation tests are nice. Can you point me to where successful changefeeds with headers are tested? Having trouble.

Reply:
TestChangefeedHeadersJSONVals - 
https://github.com/asg0451/cockroach/blob/022b52ab1a32e371773a024f21357e571c1d82d0/pkg/ccl/changefeedccl/changefeed_test.go#L10015